### PR TITLE
DM-1421: Updated device details page

### PR DESCRIPTION
**Pull request by sourajitsaha17**
The main changes are related to:
1. Measurement description with example
2. Updated some screenshots in the Controls tab since the "status" buttons changed to drop down
3. Updated screenshot from Shell tab since "View history" button is removed
4. Updated screenshot from "Predefined shell commands" - lots of changes there

### CODE REVIEW

**Comment by dafkk** [comment]
Hi, I also didn't know it at the time but images need to be smaller than we usually create. There is a whole doc explaining how to take screenshots - https://sagportal.sharepoint.com/:w:/s/IoTAnalyticsRnDOps/EW1CY4uNz2JFghnAdCXP5AUBUksbEAlsjzmXf1hY82-VKw?e=y6qgmV.

What I mainly did in my PRs, I fix the resolution to 72 and matched the height with the height of the previous image.

**Comment by sourajitsaha17** [comment]
> Hi, I also didn't know it at the time but images need to be smaller than we usually create. There is a whole doc explaining how to take screenshots - https://sagportal.sharepoint.com/:w:/s/IoTAnalyticsRnDOps/EW1CY4uNz2JFghnAdCXP5AUBUksbEAlsjzmXf1hY82-VKw?e=y6qgmV.
> 
> What I mainly did in my PRs, I fix the resolution to 72 and matched the height with the height of the previous image.

Thanks @dafkk I'll change the resolutions.

**Comment by rssag** [comment]
which releases does this apply to?

**Comment by sourajitsaha17** [comment]
this review task is for the 10.15 release

File: content/users-guide/device-management-bundle/device-details.md [hunk 1]
```diff
@@ -321,11 +321,12 @@ Hover over a row and click the delete icon, to delete the log information.
...
 
-The screenshot below, for example, shows a chart for motion measurement including graphs for acceleration in the three dimensions, and a chart with modem statistics in the form of signal strength and bit error rate.
+The screenshot below, for example, shows a chart for temperature measurement with two different series.
 
 ![Measurements](/images/users-guide/DeviceManagement/devmgmt-devices-measurements.png)
 
-If a chart contains graphs with different units, one Y-axis is rendered per unit. In the example above, motion measurements consist of three parameters with unit "meter per square second", so only one axis is rendered. Modem statistics consist of signal strength in decibel milliwatts and bit error rate in percent, so one axis is rendered for each graph.
+If a chart contains measurements with different series, one Y-axis is rendered per series. In the example above, temperature measurements consist of two parameters with series T and K (representing C and K as units), so two axes are rendered.
```
Comment (Oezge-Akin) [inline content/users-guide/device-management-bundle/device-details.md:328]: Can you do sensor1 and sensor2 for the series and same unit? 
It looks a bit strange like is it suppose to be a conversion or different sensor giving temp with different units.

**Review by torsSAG — APPROVED** [review]

**Comment by sourajitsaha17** [comment]
Reduced the resolution to 72 and also reduced the image sizes maintaining the aspect ratio

**Review by Oezge-Akin — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Review by l2p- — APPROVED** [review]

**Pull request merged**